TradeItIosTicketSDK2/TradeItDeviceManager.swift

```diff
@@ -79,4 +79,48 @@ class TradeItDeviceManager {
         )
     }
     
+    static func isDeviceJailBroken() -> Bool {
+        guard TARGET_IPHONE_SIMULATOR != 1 else {
+            return false
+        }
+        
+        // Check 1 : existence of files that are common for jailbroken devices
+        if FileManager.default.fileExists(atPath: "/Applications/Cydia.app")
+            || FileManager.default.fileExists(atPath: "/Applications/Cydia.app")
+            || FileManager.default.fileExists(atPath: "/Applications/blackra1n.app")
+            || FileManager.default.fileExists(atPath: "/Applications/FakeCarrier.app")
+            || FileManager.default.fileExists(atPath: "/Applications/Icy.app")
+            || FileManager.default.fileExists(atPath: "/Applications/IntelliScreen.app")
+            || FileManager.default.fileExists(atPath: "/Applications/MxTube.app")
+            || FileManager.default.fileExists(atPath: "/Applications/RockApp.app")
+            || FileManager.default.fileExists(atPath: "/Applications/SBSettings.app")
+            || FileManager.default.fileExists(atPath: "/Applications/WinterBoard.app")
+            || FileManager.default.fileExists(atPath: "/Library/MobileSubstrate/MobileSubstrate.dylib")
+            || FileManager.default.fileExists(atPath: "/Library/MobileSubstrate/DynamicLibraries/LiveClock.plist")
+            || FileManager.default.fileExists(atPath: "/Library/MobileSubstrate/DynamicLibraries/Veency.plist")
+            || FileManager.default.fileExists(atPath: "/private/var/lib/apt")
+            || FileManager.default.fileExists(atPath: "/private/var/lib/cydia")
+            || FileManager.default.fileExists(atPath: "/private/var/mobile/Library/SBSettings/Themes")
+            || FileManager.default.fileExists(atPath: "/private/var/stash")
+            || FileManager.default.fileExists(atPath: "/private/var/tmp/cydia.log")
+            || FileManager.default.fileExists(atPath: "/System/Library/LaunchDaemons/com.ikey.bbot.plist")
+            || FileManager.default.fileExists(atPath: "/System/Library/LaunchDaemons/com.saurik.Cydia.Startup.plist")
+            || FileManager.default.fileExists(atPath: "/bin/bash")
+            || FileManager.default.fileExists(atPath: "/usr/bin/sshd")
+            || FileManager.default.fileExists(atPath: "/etc/apt")
+            || FileManager.default.fileExists(atPath: "/usr/libexec/sftp-server")
+            || FileManager.default.fileExists(atPath: "/usr/sbin/sshd")
```

Comment:
Feels weird to call `fileExists` so verbosely every time. Can you add a private static array of these values:
```
if JAIL_BREAK_FILES.any(FileManager.default.fileExists) || UIApplication.shared.canOpenURL(URL(string:"cydia://package/com.example.package")!) {
  return true
}
```

Reply:
There doesn't seem to be an `any` method.
Is this ok to do that  : `!JAIL_BREAK_FILES.filter(FileManager.default.fileExists).isEmpty`